test/test_helper.rb

```diff
@@ -1,5 +1,3 @@
-lib_dir = File.dirname(__FILE__) + '/../lib'
-
+require 'bundler/setup'
```

Comment:
Sorry, personally I do not want to change this lines. 
I want to set `require 'debug_inspector'` explicitly in this file.

The reason there is `debug_inspector`'s RPM package in Fedora (Linux distribution) Project.
I am running this test files without Bundler.
So, I want to consider the case without Bundler.

What is harder to you?
If it is just typing, you can set alias like this.

```
alias bi='bundle install --path vendor/bundle'
alias ber='bundle exec rake'
```

Type `ber test` instead of `bundle exec rake test`.



There are many gem packages that we need to type `bundle exec rake`.

If the author @banister likes it, I just follow with him.